From a864b2b12df2f1a4edee3a44db491babf0756a2b Mon Sep 17 00:00:00 2001
From: Tam Nguyen <tam.nguyen.xa@renesas.com>
Date: Thu, 22 May 2025 00:01:51 +0700
Subject: [PATCH 39/52] arm64: dts: renesas: r8a78000: Add Ethernet PCS node

Add support for the Ethernet PCS controller on the R-Car X5H (R8A78000) SoC
by adding its device tree node.

Signed-off-by: Tam Nguyen <tam.nguyen.xa@renesas.com>
Signed-off-by: Hai Pham <hai.pham.ud@renesas.com>
---
 arch/arm/dts/r8a78000.dtsi | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/arch/arm/dts/r8a78000.dtsi b/arch/arm/dts/r8a78000.dtsi
index b7e7ec860cb..2601ed59bd2 100644
--- a/arch/arm/dts/r8a78000.dtsi
+++ b/arch/arm/dts/r8a78000.dtsi
@@ -384,6 +384,13 @@
 			status = "disabled";
 		};
 
+		eth_pcs: phy@c9c50000 {
+			compatible = "renesas,r8a78000-ether-pcs";
+			reg = <0 0xc9c50000 0 0x4000>;
+			#phy-cells = <1>;
+			status = "disabled";
+		};
+
 		sram: sram@c1000000 {
 			compatible = "arm,rcar-sram-ns", "mmio-sram";
 			reg = <0x0 0xc1000000 0x0 0x80000>;
-- 
2.34.1

